android/src/main/java/com/createthumbnail/CreateThumbnailModule.java

```diff
@@ -140,7 +140,7 @@ private static File createDirIfNotExists(String path) {
     private static Bitmap getBitmapAtTime(Context context, String filePath, int time, Map headers) {
         MediaMetadataRetriever retriever = new MediaMetadataRetriever();
         if (URLUtil.isFileUrl(filePath)) {
-            retriever.setDataSource(Uri.decode(filePath).replace("file://", ""));
+            retriever.setDataSource(filePath.replace("file://", ""));
```

Comment:
This `decode` method was deprecated in API 15 https://developer.android.com/reference/java/net/URLDecoder#decode(java.lang.String)
Can you check once with the encoding?
```suggestion
            retriever.setDataSource(Uri.decode(filePath, "UTF-8").replace("file://", ""));
```

Reply:
@souvik-ghosh Just pushed a commit using `UTF-8` as the encoding.

I had to use `URLDecoder` for that effect instead of `Uri`